meson.options

```diff
@@ -0,0 +1,10 @@
+option('libsystemd', type : 'boolean')
+option('libudev', type : 'boolean')
+option('libusb', type : 'boolean', value : false)
+option('polkit', type : 'boolean')
+option('usb', type : 'boolean')
+option('serial', type : 'boolean', value : false)
+option('systemdsystemunit', type : 'string', value : '/usr/lib/systemd/system')
+option('serialconfdir', type : 'string', value : '/etc/reader.conf.d')
+option('usbdropdir', type : 'string', value : '/usr/lib/pcsc/drivers')
+option('ipcdir', type : 'string', value : '/run/pcscd')
```

Comment:
The options support optional `description` key, which I believe would be useful for some human-readable description of the particular option.

Reply:
Done